Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/HandlesBase.cs

```diff
@@ -80,7 +80,7 @@ protected void UpdateColliderBounds()
         protected abstract void UpdateColliderBounds(Transform handle, Vector3 visualSize);
         protected abstract void RecreateVisuals();
 
-        protected void ResetHandles()
+        protected virtual void ResetHandles()
```

Comment:
Prepping for extensible/subclassed handles.

Reply:
Going forward, I'd recommend against doing prep work for future PRs that increase/change the public API surface.

There are any number of things that could happen that would invalidate a future change, which would leave code in this state (i.e. virtual but nothing extending). 

If the very next PR is going to be using this then that's fine - if we're a couple of PRs out then I would recommend removing these changes from this because they go directly to mainline.

Reply:
@wiwei Downgrading back to `private` for all changed properties.